docs/sql/window.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: default
+title: Window Functions
+parent: SQL
+nav_order: 11
+---
+
+# Window Functions
+
+A window function performs a calculation across a frame of data rows around the current row and finds a result for each row.
+
+`PARTITION BY` and `ORDER BY` define the frame of data over which the calculation is made.
+
+You can use window functions in the following three categories:
+
+1. Aggregate Functions: `COUNT()`, `MIN()`, `MAX()`, `AVG()`, and `SUM()`.
+2. Ranking Functions: `ROW_NUMBER()`, `RANK()`, `DENSE_RANK()`, `PERCENT_RANK()`, and `NTILE()`.
+3. Analytic Functions: `CUME_DIST()`, `LAG()`, and `LEAD()`.
+
+The syntax of a window function is as follows:
+
+```sql
+function_name (expression [, expression...])
+  OVER (
+    PARTITION BY expression [, expression...]
+    ORDER BY expression [ASC | DESC] [, ...]
+    )
+```
+
+The `PARTITION BY` and `ORDER BY` clauses are optional.
+{: .note }
+
+To use window functions, enable the new SQL engine:
+
+```json
+PUT _cluster/settings
+{
+ "transient": {
```

Comment:
I think persistent is by far the better option here, but that's not a blocker to merging.

Reply:
Changed. Yep, part of 1.11. Merged.